packages/spec/src/index.ts

```diff
@@ -1,6 +1,106 @@
-import type * as v1 from "./v1.ts";
+export declare namespace v1 {
```

Comment:
We could also remove the namespace and add it as needed when we actually release a v2 at some point in the future, so as not to be too picky about the versioning structure from the beginning on.

Reply:
I like that this approach conveys our future intent for version management and doesn't feel too onerous to deal with.

Reply:
Agreed, makes any hypothetical future migration path feel more incremental, which is good.

Reply:
Ok 👍